src/components/mapexplorer.js

```diff
@@ -380,13 +382,31 @@ function MapExplorer({
 
   const {name, lastupdatedtime} = currentHoveredRegion;
 
+  function getCurrentDate() {
+    const date = new Date();
+    let getMonth = String(date.getMonth() + 1);
+    if (getMonth.length === 1) {
+      getMonth = '0' + getMonth;
+    }
```

Comment:
This can be done in a single step using the padStart() in JS, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart

Reply:
Thanks, updated the code.